src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java

```diff
@@ -46,26 +49,32 @@
   private final ConnectionProvider connectionProvider;
   private final ConnectorContext context;
   private final CountDownLatch shutdownLatch;
+  private final long startupMs;
   private final long pollMs;
   private final Set<String> whitelist;
   private final Set<String> blacklist;
   private final AtomicReference<List<TableId>> tables;
+  private final Time time;
 
   public TableMonitorThread(DatabaseDialect dialect,
       ConnectionProvider connectionProvider,
       ConnectorContext context,
+      long startupMs,
       long pollMs,
       Set<String> whitelist,
-      Set<String> blacklist
+      Set<String> blacklist,
+      Time time
   ) {
     this.dialect = dialect;
     this.connectionProvider = connectionProvider;
     this.context = context;
     this.shutdownLatch = new CountDownLatch(1);
+    this.startupMs = startupMs;
```

Comment:
can we rely on existing pollms config for this?

Reply:
or use a constant for now before we see compelling reasons to expose a configuration

Reply:
I made a conscious decision to separate the two as the time spent here waiting on startup is blocking the herder thread. The polling interval is specified by `table.poll.interval.ms` , which defaults to a minute. I don't think many users would change this. And, I did not want to be blocking the herder thread for up to a minute in the default config. Conceptually too, these are very different time intervals, so it makes sense you might want to adjust them separately.